app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -17,7 +17,7 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
   require_nested :Template
   require_nested :Vm
 
-  OrchestrationTemplate.register_eligible_manager(self)
+  self::OrchestrationTemplate.register_eligible_manager(self)
```

Comment:
why is this change needed?

Reply:
There was an issue, probably related to the rails dev env lazy code loading.
Rails was looking at the top level constant first, and there is no such method.
However, I forgot how to reproduce it now, so, reverting.